src/alignment/configs.py

```diff
@@ -244,102 +245,15 @@ class SFTConfig(transformers.TrainingArguments):
     dataset_kwargs: Optional[Dict[str, Any]] = field(
         default=None, metadata={"help": "Dataset kwargs for the SFTTrainer"}
     )
-    max_seq_length: Optional[int] = field(
-        default=None,
-        metadata={"help": ("Used by TRL for reward model training, which tries to read this parameter in init.")},
-    )
-    logging_first_step: bool = field(
-        default=True,
-        metadata={"help": ("Whether to log and evaluate the first global_step or not.")},
-    )
-    optim: Optional[str] = field(default="adamw_torch")
 
 
 @dataclass
-class DPOConfig(transformers.TrainingArguments):
+class DPOConfig(trl.DPOConfig):
     """
     Arguments related to the DPO training process itself. For all parameters, see: https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/trainer#transformers.TrainingArguments
     """
 
-    beta: Optional[float] = field(
-        default=0.1,
-        metadata={"help": "The beta factor in DPO loss. Higher beta means less divergence from the initial policy."},
-    )
     hub_model_revision: Optional[str] = field(
         default="main",
         metadata={"help": ("The Hub model branch to push the model to.")},
     )
```

Comment:
As per the comments below and based on https://github.com/huggingface/trl/blob/main/trl/trainer/dpo_config.py, this configuration should be as follows in order to keep the consistency with the previous one:

```suggestion
    hub_model_revision: Optional[str] = field(
        default="main",
        metadata={"help": ("The Hub model branch to push the model to.")},
    )
    logging_first_step: bool = field(
        default=True,
        metadata={"help": ("Whether to log and evaluate the first global_step or not.")},
    )
    optim: Optional[str] = field(default="rmsprop")
    remove_unused_columns: bool = field(default=False)
```